packages/backend/src/graphql/resolvers/incoming_payment.ts

```diff
@@ -22,18 +21,21 @@ export const getAccountIncomingPayments: AccountResolvers<ApolloContext>['incomi
   )
 
   return {
-    edges: incomingPayments.map((incomingPayment: IncomingPayment) => ({
-      cursor: incomingPayment.id,
-      node: {
-        ...incomingPayment,
-        receivedAmount: {
-          assetCode: incomingPayment.asset.code,
-          assetScale: incomingPayment.asset.scale
-        },
-        expiresAt: incomingPayment.expiresAt.toISOString(),
-        createdAt: incomingPayment.createdAt?.toISOString()
+    edges: incomingPayments.map((incomingPayment: IncomingPayment) => {
```

Comment:
I think this can be
```
    edges: incomingPayments.map((incomingPayment: IncomingPayment) => ({
      cursor: incomingPayment.id,
      node: {
        ...incomingPayment,
        receivedAmount: incomingPayment.receivedAmount,
        expiresAt: incomingPayment.expiresAt.toISOString(),
        createdAt: incomingPayment.createdAt.toISOString()
      }
    }))
```